### DISCUSSION TITLE
Bluetooth Passthrough

### DISCUSSION THREAD

**Discussion by kenfink** [AppGround-io/bluetooth-keyboard-and-mouse-support#332] [Ideas]
I have a physical Bluetooth keyboard and mouse that I have paired with my phone, for use on the go.  Each device can only pair to one device at a time.

I have other systems that I also want to use the same keyboard and mouse with (Laptop, Oculus Quest, etc..).  I'll use the Quest as an example, but this applies to any generic connection.  Currently, I have to unpair both devices from my phone, then re-pair them to the Quest every time, then re-pair them to my phone later.  While multi-device Bluetooth keyboards exist, most mice are still single-paired-device.

Realizing I'll have my phone nearby every time I'm using the Quest, I'd love to pair this app with the Quest (it actually already is, so I can use my phone as a trackpad).  I'd like to be able to put the app in "passthrough mode" so when I type on my keyboard or move the mouse it controls the secondary device.

Essentially, capture the mouse and keyboard input coming into the android device, and then re-broadcast them through the Bluetooth Keyboard and Mouse App stack. 

Thanks for writing a fantastic and useful app!

**Comment by WsQ82** [comment]
I have a similar but different need. Id like it to pass through my Galileo G8 controller so I can use my phone with it as trackpad/additional buttons.
